docs/ref/models/fields.txt

```diff
@@ -1116,7 +1116,7 @@ Django also defines a set of fields that represent relations.
 ``ForeignKey``
 --------------
 
-.. class:: ForeignKey(othermodel, [**options])
+.. class:: ForeignKey(othermodel, on_delete, [**options])
```

Comment:
Certainly for reusable apps, but for a lot of projects backwards-compatibility is not a concern at all, and for a required argument to all FKs, whose purpose is made fairly evident by the argument itself, I think quite a few people (possibly myself included) will prefer to save the extra characters. So I'd be in favor of allowing it to be passed positionally.